packages/compass-assistant/src/components/assistant-chat.tsx

```diff
@@ -371,14 +368,26 @@ export const AssistantChat: React.FunctionComponent<AssistantChatProps> = ({
               documentation right in your window.
             </div>
           )}
-          <InputBar
-            data-testid="assistant-chat-input"
-            onMessageSend={handleMessageSend}
-            state={status === 'submitted' ? 'loading' : undefined}
-            textareaProps={{
-              placeholder: 'Ask MongoDB Assistant a question',
-            }}
-          />
+          <div className={inputBarStyleFixes}>
+            <InputBar
+              data-testid="assistant-chat-input"
+              onMessageSend={handleMessageSend}
+              state={status === 'submitted' ? 'loading' : undefined}
+              textareaProps={{
+                placeholder: 'Ask a question',
+              }}
+            />
+          </div>
+          <DisclaimerText className={disclaimerTextStyles}>
+            AI can make mistakes. Review for accuracy. See our{' '}
+            <Link
+              hideExternalIcon={false}
+              href={GEN_AI_FAQ_LINK}
+              target="_blank"
+            >
+              Learn more
+            </Link>
+          </DisclaimerText>
```

Comment:
"See our Learn more"? This should either be "See our FAQ" or just "Learn more", right?

Reply:
Yeah this ended up being a frankenstein of the two 😅